.github/pull_request_template.md

```diff
@@ -9,14 +9,20 @@ This PR ...
 - [ ] Ran `quickTest all`?
 - [ ] Downloaded a file from http://localhost:7071/api/download
 - [ ] Updated the release notes? 
-- [ ] Added a test?
-- [ ] Did you check for sensitive data, and remove any?
+- [ ] Added tests?
+- [ ] Did you check for sensitive data, and remove any? 
+- [ ] Does logging contain sensitive data?  
```

Comment:
PRs for code changes are a good point to add a formal security review process, but I think that's a longer checklist than you could fit here.

Reply:
+1 to the log mention